### PR TITLE
Fixing tests

### PR DESCRIPTION
**Pull request by loudnate**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,11 +2,8 @@ language: objective-c
...
 xcode_sdk: 
     - iphonesimulator9.3
-    - iphonesimulator9.2
 xcode_project: RileyLink.xcodeproj
 xcode_scheme: 
     - RileyLink
 script:
-    # We need to enumerate these manually until xctool is fixed
     - xcodebuild -project RileyLink.xcodeproj -scheme RileyLink -sdk iphonesimulator9.3 test
-    - xcodebuild -project RileyLink.xcodeproj -scheme RileyLink -sdk iphonesimulator9.2 test
```
Comment (loudnate) [inline .travis.yml:12]: @ps2 I'm not sure how to run these tests against the 9.2 simulator while still using xcode7.3 (Swift 2.2)

File: MinimedKit/PumpEvents/Sara6EPumpEvent.swift [hunk 2]
```diff
@@ -27,7 +27,7 @@ public class Sara6EPumpEvent: TimestampedPumpEvent {
     
     let dateComponents = TimeFormat.parse2ByteDate(availableData, offset: 1)
     validDateStr = String(format: "%04d-%02d-%02d", dateComponents.year, dateComponents.month, dateComponents.day)
-    timestamp = TimeFormat.midnightForDate(dateComponents)
+    timestamp = dateComponents
```
Comment (ps2) [inline MinimedKit/PumpEvents/Sara6EPumpEvent.swift:30]: The dictionary representation should still use the midnightForDate value, I think.

Comment (loudnate) [inline MinimedKit/PumpEvents/Sara6EPumpEvent.swift:30]: I agree it should have one day added, but it shouldn't use the local time zone midnight. Want me to change that?

Comment (loudnate) [inline MinimedKit/PumpEvents/Sara6EPumpEvent.swift:30]: Here's a quick fix for now, until we can root out all uses of `localTimeZone`.

**Comment by ps2** [comment]
:+1:

**Pull request merged**